assets/scss/components/stencil/productView/_productView.scss

```diff
@@ -22,25 +22,38 @@
     align-items: center;
     display: flex;
     justify-content: center;
-    margin: 0;
+    margin: auto;
     position: relative;
 
     @include breakpoint("medium") {
         min-height: 366px;
         min-width: inherit;
     }
 
-    img {
-        width: 100%;
+    + .productView-thumbnails {
+        margin-top: spacing("half");
     }
+}
+
+.productView-img-container {
+    position: relative;
+    margin: auto;
+    max-width: get-width(stencilString('product_size'));
+    width: 100%;
 
-    .productView-image--default {
+    img {
+        @include lazy-loaded-img;
+        max-height: 100%;
         width: auto;
     }
+}
 
-    + .productView-thumbnails {
-        margin-top: spacing("half");
-    }
+.productView-img-container:after {
+    content: '';
```

Comment:
just a helpful pointer - a few other places lend themselves well to using mixins